src/video_core/renderer_vulkan/vk_buffer_cache.cpp

```diff
@@ -182,6 +182,9 @@ void BufferCacheRuntime::BindQuadArrayIndexBuffer(u32 first, u32 count) {
 
 void BufferCacheRuntime::BindVertexBuffer(u32 index, VkBuffer buffer, u32 offset, u32 size,
                                           u32 stride) {
+    [[unlikely]] if (index >= device.GetMaxVertexInputBindings()) {
```

Comment:
```suggestion
    if (index >= device.GetMaxVertexInputBindings()) {
```

It's preferable for this to only be used on paths where it's been benchmarked that it provides notable benefits